ckanext/dia_theme/templates/header_base.html

```diff
@@ -74,9 +74,9 @@
         {% endblock %}
         <div class="container">
             <button data-target=".nav-collapse" data-toggle="collapse" class="btn btn-navbar" type="button" role="presentation" tabindex="-1">
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
+                <span class="fa fa-bar"></span>
+                <span class="fa fa-bar"></span>
+                <span class="fa fa-bar"></span>
```

Comment:
I don't think there's an fa-bar, but icon-bar should still work

Reply:
Nice find.

I discovered a fa-bars icon that we can use instead